spec/integration/api/auth_spec.rb

```diff
@@ -216,6 +216,7 @@ module Vault
     describe "#aws_iam" do
       before(:context) do
         vault_test_client.sys.enable_auth("aws", "aws", nil)
+        vault_test_client.sys.put_auth_tune("aws", "iam_server_id_header_value" => "iam_header_canary")
```

Comment:
I think the `.sys.put_auth_tune` here isn't right. The configuration of the header value is done via the [`auth/aws/config/client`](https://www.vaultproject.io/api/auth/aws/index.html#iam_server_id_header_value) API endpoint, not via a mount-tune call.